### PR TITLE
Better error catching, v0datalink

### PR DESCRIPTION
**Pull request by ddobrigk**

### CODE REVIEW

File: PWGLF/TableProducer/strangenessbuilder.cxx [hunk 1]
```diff
@@ -394,6 +395,7 @@ struct strangenessBuilder {
     } catch (...) {
       registry.fill(HIST("hCaughtExceptions"), 0.5f);
       LOG(error) << "Exception caught in DCA fitter process call!";
+      return false;
```
Comment (njacazio) [inline PWGLF/TableProducer/strangenessbuilder.cxx:398]: Ah! now I get it

Comment (njacazio) [reply PWGLF/TableProducer/strangenessbuilder.cxx:398]: Should we align also the other builders?

Comment (ddobrigk) [reply PWGLF/TableProducer/strangenessbuilder.cxx:398]: Maybe! Done! Thanks! 👍

**Review by njacazio — APPROVED** [review]

**Comment by njacazio** [comment]
Thanks!

**Pull request merged**